src/components/form.jsx

```diff
@@ -143,22 +144,40 @@ const Form = () => {
             </button>
           </form>
 
+          <style>
```

Comment:
oof uhm... this project uses tailwind, and I'm sure there is some way to apply these classes to the required button using the tailwind way. Why go this route? I like the output, but this in the middle of the code just makes it look out of place.